pkg/synthetictests/kubelet.go

```diff
@@ -16,6 +16,11 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	probeTimeoutEventThreshold                 = 5
+	probeTimeoutConfigOperatorMessageRegExpStr = "reason/ReadinessFailed.*Get.*healthz.*net/http.*request canceled while waiting for connection.*Client.Timeout exceeded"
```

Comment:
I see you moved to using an `allowedRepeatedEventFns` which makes sense, but I would rename this `probeTimeoutConfigOperatorMessageRegExpStr` to `probeTimeoutMessageRegExpStr` since nothing in this string is specific to the config operator

Reply:
Good point; change pushed.